app.js

```diff
@@ -1,16 +1,147 @@
 /* Imports */
 
 /* State */
+let gameState = 'choice'; //choose or results
+let choice = ''; //stone, cloth, or scythe
+const computerArray = ['stone', 'cloth', 'scythe'];
+let computerChoice = getRandomItem(computerArray);
+
+let wins = 0;
+let total = 0;
 
 /* Actions */
-function loadPage() {}
+function loadPage() {
+    scoreboardMath();
+    displayResults();
+    winConditions();
+}
+
+function playAgain() {
+    gameState = 'choice';
```

Comment:
This is setting the state back to choice, but to replay, we have to also think about the rest of the state; we would want all of that to be set to the **_default state_**, right? (besides the scoreboard state, of course) So how would you accomplish that?